embulk-jruby-extension/test/mri/date/test_date_strptime_simple.rb

```diff
@@ -0,0 +1,319 @@
+#
+# how to use
+# $ jruby -J-Xbootclasspath/a:embulk-jruby-extension/build/libs/embulk-jruby-extension-0.6.7.jar embulk-jruby-extension/test/mri/date/test_date_strptime_simple.rb
+#
+require 'test/unit'
+require 'date'
+
+require 'java'
+
+class TestDateStrptime < Test::Unit::TestCase
+
+  STRFTIME_2001_02_03 = {
```

Comment:
Did you change symbolized map to string map like `{"wday"=>6}` to `{:wday=>6}`? I think it's not a good idea because it makes it very difficult to merge upstream changes. We don't want to create a fork.